internal/services/automanage/migration/configuration_v0_to_v1.go

```diff
@@ -0,0 +1,338 @@
+package migration
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/automanage/2022-05-04/configurationprofiles"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+var _ pluginsdk.StateUpgrade = ConfigurationV0ToV1{}
+
+type ConfigurationV0ToV1 struct {
+}
+
+func (ConfigurationV0ToV1) Schema() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+
+		"resource_group_name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+
+		"location": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+
+		// "Antimalware/Enable": boolean, true if block exists
+		// "Antimalware/EnableRealTimeProtection": boolean,
+		// "Antimalware/RunScheduledScan": boolean,
+		// "Antimalware/ScanType": string ("Quick", "Full"),
+		// "Antimalware/ScanDay": int (0-8) Ex: 0 - daily, 1 - Sunday, 2 - Monday, .... 7 - Saturday, 8 - Disabled,
+		// "Antimalware/ScanTimeInMinutes": int (0 - 1440),
+		// "Antimalware/Exclusions/Extensions": string (extensions separated by semicolon. Ex: ".ext1;.ext2"),
+		// "Antimalware/Exclusions/Paths": string (Paths separated by semicolon. Ex: "c:\excluded-path-1;c:\excluded-path-2"),
+		// "Antimalware/Exclusions/Processes": string (Processes separated by semicolon. Ex: "proc1.exe;proc2.exe"),
```

Comment:
do we need this comment here for the state migration?

Reply:
Good shout, removed.